core/deployment/src/main/java/io/quarkus/deployment/logging/LoggingResourceProcessor.java

```diff
@@ -76,14 +78,25 @@ void miscSetup(
                 new ServiceProviderBuildItem(EmbeddedConfigurator.class.getName(), InitialConfigurator.class.getName()));
     }
 
-    @BuildStep
+    @BuildStep(loadsApplicationClasses = true)
```

Comment:
Did I mention that this needs to change? If I did, sorry, I was wrong. This should not be needed, only the other `loadsApplicationClasses = true` you added

Reply:
But it was not loading resource. When I changed this too. It did

Reply:
Oh wait , did you mean , `loadApplicationClassTrue` in banner extension and not on LoggingResourceProcessor

Reply:
@geoand  I checked , I have to add `loadApplicationClasses= true` to both places. Otherwise It is not working
Otherwise it get stucks at this line 
```
[INFO] --- quarkus-maven-plugin:999-SNAPSHOT:dev (default-cli) @ banner-test ---
Listening for transport dt_socket at address: 5005
```

No output further

Reply:
I'll check again and let you know, but I really don't think it's necessary

Reply:
I double-checked a test application and indeed this is not needed. 

The reason why it's needed in the other build step you added is because you are attempting you read a classpath resource inside a build step - a resource that was added by the user. The default Thread Context ClassLoader that is used when build steps are run (which Quarkus calls the `Augmentation ClassLoader`) does not have access to the application classes and resources.
When using `loadApplicationClasses` then the Thread Context ClassLoader that is set when the build step is run (is what Quarkus calls the `Deployment ClassLoader`) and this does have access to application classes and resources.

The logging processor you have changed here has nothing to do with loading the resource.

Reply:
I removed it from LoggingResourceProcessor , but then it doesn't work . As I output stuck

Reply:
You must be doing something wrong. I tried it multiple times

Reply:
You mean on your side banner is working ?

Reply:
Yes, when I use `@BuildStep(loadsApplicationClasses = true)` in `BannerProcessor` only and leave this one without it (like it was before the change), then a custom banner file placed inside `src/main/resources` and configured in `application.properties`, works just fine

Reply:
Ok its working now.
I probably erased `@BuildStep` annotation itself instead of removing `loadApplicationClasses = true` only from LoggingResourceProcessor. :sweat:

Reply:
Great!